client-guide/agile-workflows.html.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Agile workflows
+---
+
+The single biggest problem we have seen in development teams is the lack of an established
+philosophy and process for managing the day-to-day work of the developers and ensuring they are
+delivering value to the business without requiring a constant back and forth. Bureaucracy is
+satisfied in the name of speed, but this leads to teams that only perform at a fraction of their
+potential, often leaving stakeholders dissatisfied.
```

Comment:
What about using `unhappy` instead of `dissatisfied`? I think it's more common but mainly because we are already using `satisfied` in the same sentence.

Reply:
@kennyadsl good catch, fixing!

Reply:
Also realized "satisfied" was a mistake, should have been "sacrificed". I was probably already thinking how to end the sentence. 😄